web/packages/teleterm/package.json

```diff
@@ -44,8 +44,8 @@
     "@types/whatwg-url": "^11.0.5",
     "@xterm/xterm": "^5.5.0",
     "@xterm/addon-fit": "^0.10.0",
-    "electron": "31.1.0",
-    "electron-builder": "^25.0.1",
+    "electron": "31.3.1",
+    "electron-builder": "^25.0.3",
```

Comment:
@avatus Is there a way to put electron and electron-builder into a separate group? These two require a manual tag build to be properly tested.

I'm running a tag build to test these changes. https://github.com/gravitational/teleport.e/actions/runs/10196686755

Reply:
Yes, we can update the config to exclude specific patterns from the grouping. I'll open a PR but will wait to merge before the next patch cycle (to prevent spam)

```md
    groups:
      ui:
        update-types:
          - "minor"
          - "patch"
        exclude-patterns:
          - "electron*"
```

Reply:
https://github.com/gravitational/teleport/pull/44952